packages/material-ui/src/TablePagination/TablePagination.d.ts

```diff
@@ -17,11 +17,13 @@ export interface TablePaginationTypeMap<P, D extends React.ElementType> {
   props: P &
     TablePaginationBaseProps & {
       ActionsComponent?: React.ElementType<TablePaginationActionsProps>;
+      backIconButtonText?: string;
       backIconButtonProps?: Partial<IconButtonProps>;
       count: number;
       labelDisplayedRows?: (paginationInfo: LabelDisplayedRowsArgs) => React.ReactNode;
-      labelRowsPerPage?: React.ReactNode;
+      labelRowsPerPage?: string;
```

Comment:
Nice, hopefully, fixed in #20488.

Reply:
Thanks for the link. This PR fixes that issue.

Reply:
Right, it makes it a feature request instead of a bug :)